staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -401,7 +408,7 @@ func (p *patcher) patchResource(ctx context.Context, scope RequestScope) (runtim
 	p.updatedObjectInfo = rest.DefaultUpdatedObjectInfo(nil, p.applyPatch, p.applyAdmission)
 	result, err := finishRequest(p.timeout, func() (runtime.Object, error) {
 		// TODO: Pass in UpdateOptions to override UpdateStrategy.AllowUpdateOnCreate
-		updateObject, created, updateErr := p.restPatcher.Update(ctx, p.name, p.updatedObjectInfo, p.createValidation, p.updateValidation, false)
+		updateObject, created, updateErr := p.restPatcher.Update(ctx, p.name, p.updatedObjectInfo, p.createValidation, p.updateValidation, p.forceAllowCreate)
```

Comment:
@jennybuckley Could you sync with #65572?  Checking `CREATE` operation in `p.createValidation`?😃

Reply:
Thanks for the review!

I think #65572 is to fix the admission called in create on update, but this PR is only concerned with create on patch. The server side apply feature branch, which this PR is being made to, already has a PR merged to it (#64892) which gets the right validating and mutating admission called. The staticAdmissionAttributes for create are set [here](https://github.com/jennybuckley/kubernetes/blob/a2a02afe4664cbfb5334c008304254a559f4ec5f/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go#L113), and then the p.createValidation is built using those attributes [here](https://github.com/jennybuckley/kubernetes/blob/a2a02afe4664cbfb5334c008304254a559f4ec5f/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go#L127)